tasks/altsrc.conf

```diff
@@ -0,0 +1,32 @@
+[altsrc]
+#pagure api
+# pagure_api_key_file = ./config/pagure_test/pagure.key
+# pagure_repo_init_api = https://git.dev.centos.org/api/0/new
+pagure_repo_init_api = https://git.dev.centos.org/api/0/new
```

Comment:
Can you transform altsrc.conf as a .j2 jinja2 template ? 
That way , it can be templatized for every environment and no need for a "static" file. (you can set defaults in defaults/main.yml too)